app/views/pets/edit.html.erb

```diff
@@ -1,14 +1,4 @@
-<div class="background">
-  <img src="https://wallpaperaccess.com/full/783057.jpg" alt="kitten">
-</div>
-
 <div class="content">
-  <div class="main_menu">
-    <a href="/">Main</a>
-    <a href="/shelters">Shelters</a>
-    <a href="/shelters/new">New Shelter</a>
-    <a href="/pets">Pets</a>
-  </div>
 
```

Comment:
this is the same for all views. now the navbar and background image are in the views/layouts/application

Reply:
beautiful